aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -505,6 +505,9 @@ module aptos_token::token {
         id: TokenId,
         amount: u64,
     ): Token acquires TokenStore {
+        // Always make sure the account has sufficient tokens to withdraw.
+        assert!(balance_of(account_addr, id) >= amount, EINSUFFICIENT_BALANCE);
```

Comment:
Please wrap the error code with error::invalid_argument(EINSUFFICIENT_BALANCE) or whichever function in error that's appropriate